test-files/cases/attribute/unifying/building/building-3022-duplicate-relation-member-ways-1/Expected.osm

```diff
@@ -71,32 +89,37 @@
         <nd ref="-16"/>
         <nd ref="-15"/>
         <nd ref="-6"/>
-        <tag k="hoot:status" v="1"/>
         <tag k="building" v="yes"/>
+        <tag k="hoot:id" v="-2"/>
+        <tag k="hoot:status" v="1"/>
+        <tag k="hoot:way:node:count" v="15"/>
     </way>
     <way visible="true" id="-1" timestamp="1970-01-01T00:00:00Z" version="1">
         <nd ref="-4"/>
         <nd ref="-3"/>
         <nd ref="-2"/>
         <nd ref="-1"/>
         <nd ref="-4"/>
-        <tag k="hoot:status" v="1"/>
         <tag k="building" v="yes"/>
+        <tag k="hoot:id" v="-1"/>
+        <tag k="hoot:status" v="1"/>
+        <tag k="hoot:way:node:count" v="5"/>
     </way>
```

Comment:
Here is an example of what ways look like now for unit test output, `hoot:way:node:count` tag added and all tags sorted alphabetically.